### DISCUSSION TITLE
Can GLPI Agent version 1.7.3 support Esxi 6?

### DISCUSSION THREAD

**Discussion by thasleemnazeer** [glpi-project/glpi-agent#644] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

I'm using ESXi 6 and want to confirm if the GLPI Agent version 1.7.3 will be supported.

**Comment by g-bougard** [comment]
Yes it is.

`ESX` task is supporting ESXi v6.
